src/css/index.scss

```diff
@@ -0,0 +1,134 @@
+//public
```

Comment:
Well done Chris, I run your code on my local and the style part you did is very close to our example site.
It shows that you really pay attention on the class and I'm really happy for that.
Thanks a lot mate.

Reply:
I`ve read all the recommandations and I really appreciate that mate.
I have to admit it that the empty table rows is an terrible issue as I wrongly added them during editing, besides, the form item do should include names, it is true that detail matters.
Thanks.